### PR TITLE
Chat polish

### PR DESCRIPTION
**Pull request by Duncid**
This is actually an old PR I forgot about.
They are improvements around chat (readability, style).

### CODE REVIEW

**Review by spolu — COMMENTED** [review]
Left 2 quick questions

Tested it, otherwise LGTM

**Comment by spolu** [comment]
Screenshot for posterity
Screenshot from 2023-09-05 13-30-10

File: front/pages/w/[wId]/u/chat/[cId]/index.tsx [hunk 1]
```diff
@@ -1194,7 +1200,7 @@ export default function AppChat({
 
         {canStartConversation && (
           <>
-            <div className="flex-1">
+            <div id="MessageView" className="flex-1">
```
[the remark this answers is not in the document]
Comment (Duncid) [reply front/pages/w/[wId]/u/chat/[cId]/index.tsx:1203]: No, we don't.
However, I think we should use ids more often just to make the code more discoverable.
It's very difficult in page like "[front/pages/w/[wId]/u/chat/[cId]/index.tsx](https://github.com/dust-tt/dust/pull/1250/files/94b7fa391155e417740eff05d3ecba5aff76d3b8#diff-b4851b50e49603ac62c665b6752cbe5bbbcb981bffac7c59ba2aa39716a71075)" to find what is what.
I tend to inspect the dom to try to locate what i want to change.

File: sparkle/src/_index.ts [hunk 2]
```diff
@@ -10,8 +10,8 @@ export { SliderToggle };
 import { Tab } from "./components/Tab";
 export { Tab };
 
-import { Item } from "./components/Item";
-export { Item };
+import { Item, ItemSectionHeader, ListItem } from "./components/Item";
```
[the remark this answers is not in the document]
Comment (Duncid) [reply sparkle/src/_index.ts:13]: It's an error when i rebased.
Fixing.

File: front/pages/w/[wId]/u/chat/[cId]/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (spolu) [reply front/pages/w/[wId]/u/chat/[cId]/index.tsx:1203]: Component names should serve that purpose in general. For Chat v0 agreed that we could do a better job. But out of consistency of the codebase let's not introduce `id` without more thinking and methodology :pray:

[the remark this answers is not in the document]
Comment (Duncid) [reply front/pages/w/[wId]/u/chat/[cId]/index.tsx:1203]: Component names are not visible from inspecting in the browser are they?

**Pull request merged**